frontend/src/components/problem/ProblemDisplay.tsx

```diff
@@ -138,21 +153,24 @@ const NoMarginTopText = styled(Text)`
   margin-top: 0px;
 `;
 
-const InputTypeContainer = styled.div`
-  margin-bottom: 5px;
-`;
-
-const CancelTextButton = styled(TextButton)`
-  margin-left: 2.5px;
-  color: ${({ theme }) => theme.colors.gray};
-`;
-
 const DeleteButton = styled(PrimaryButton)`
   margin: 0 0 1rem 0;
   color: ${({ theme }) => theme.colors.red2};
   background: ${({ theme }) => theme.colors.white};
 `;
 
+const TextButtonLink = styled(TextButton)`
+  padding: 0;
+  margin-top: 10px;
+  text-decoration: underline;
+  text-align: left;
+`;
```

Comment:
Small styling suggestions: maybe move this down a few lines (like so it's shown a little farther down when no description exists), and/or also maybe make this text gray?

Reply:
I made this text the same color as the text placeholder, and moved it `20px` farther down; however, I don't think it's super visible. Any thoughts or advice here @alankbi? If you have a color or margin in mind that'd help, no problem if not.

**New Template Description Text**
<img width="1280" alt="New Template Description Text" src="https://user-images.githubusercontent.com/7987279/120545159-24d5a980-c3a3-11eb-8b98-9c359bfd5238.png">

Reply:
Is this the default gray or a slightly different color? If the default gray doesn't look good, then I'm fine with reverting it to the original color, either or

Reply:
This is the placeholder gray that is in the above Markdown setup. I'll try switching around on different grays, and seeing what looks best 👍

Reply:
I switched to the default gray as you said, that looked good!